packages/oui-pagination/_mixins.less

```diff
@@ -0,0 +1,138 @@
+@import '../oui-button/_variables';
+@import './_variables';
+
+#oui {
+  .pagination-base(
+    @button-selector,
+    @button-link-selector,
+    @button-secondary-selector,
+    @button-group-selector
+  ) {
+    #oui > .base-font();
+
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    padding: @oui-pagination-padding;
+    font-size: @oui-pagination-font-size;
+
+    /* stylelint-disable indentation */
+    .phone({
+      flex-direction: column-reverse;
+      justify-content: inherit;
+      align-items: center;
+    });
+    /* stylelint-enable indentation */
+
+    .@{button-selector}, .@{button-group-selector} {
+      margin-bottom: 0;
+    }
+
+    .@{button-link-selector} {
+      padding-left: 0;
+      padding-right: 0;
+      margin-right: rem-calc(5);
+    }
+
+    &__progress {
+      display: flex;
+      align-items: center;
+      flex-wrap: nowrap;
+    }
+
+    &__text {
+      display: inline-block;
+    }
+
+    &__selector {
+      display: flex;
+      flex-wrap: nowrap;
+      margin-left: rem-calc(20);
+
+      /* stylelint-disable indentation */
+      .phone({
+        margin-bottom: rem-calc(10);
+        margin-left: 0;
+      });
+      /* stylelint-enable indentation */
+
+      // Fix weird padding on Firefox
+      .@{button-secondary-selector} {
+        padding-top: @oui-button-hollow-padding-base;
+        padding-bottom: @oui-button-hollow-padding-base;
```

Comment:
(optionnal) We can remove this part, no longer useful at the moment.
Already provide by `.oui-button_secondary`.

Reply:
-> Fix weird padding on Firefox
If I remove that, it breaks UI on FF

Reply:
I know.

I read it and test it before adding my comment.
At the moment, I cannot reproduce that _weird_ bug.